package.json

```diff
@@ -2,7 +2,8 @@
   "private": true,
   "license": "MIT",
   "workspaces": [
-    "packages/*"
+    "packages/*",
+    "experiments"
```

Comment:
`experiments/*` will enable parallel experiments 😄

Reply:
ahhhhhh... of course! right! I mean is it really worth it? isn't it better to have them all under the same "project"? 🤔 I'm not sure, though... Anyways, go for it if you want, for sure! Run the experiment! 🙈